Consul.Test/OperatorTest.cs

```diff
@@ -123,5 +123,24 @@ public async Task Operator_AreaList()
             Assert.Equal(area.UseTLS, result.UseTLS);
             Assert.Equal(area.RetryJoin, result.RetryJoin);
         }
+        [EnterpriseOnlyFact]
+        public async Task Operator_AreaUpdate()
+        {
+            var peerDataCenter = KVTest.GenerateTestKeyName();
+            var area = new AreaRequest { PeerDatacenter = peerDataCenter, UseTLS = false, RetryJoin = null };
+            var createResult = await _client.Operator.AreaCreate(area);
+            var areaId = createResult.Response;
+
+            area = new AreaRequest { PeerDatacenter = peerDataCenter, UseTLS = true, RetryJoin = new string[] { "10.1.2.9", "10.1.2.0" } };
+            var updateResult = await _client.Operator.AreaUpdate(area, createResult.Response);
```

Comment:
```suggestion
            var updateResult = await _client.Operator.AreaUpdate(area, areaId);
```

Reply:
Assertion for `updateResult` is missing now, please add it back 😄

Reply:
I removed it because we're already using the initial createResponse areaId here: ```var updatedArea = listResult.Response.Single(x => x.ID == areaId);``` to fetch the updated query. This somehow asserts that createResult areaId is equal to updatedResult areaId, so the additional assertion is seemed redundant. I guess I was over thinking it. I have added it back now—thanks 😊

Reply:
Thanks, one last thing. Please apply the suggested change for using `areaId` instead of `createResult.Response` (If I do it I'll not be able to approve the PR).